common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
                 ],
             },
             {
-                "short_description": _("Draft a Rough Course Outline"),
+                "short_description": _("Course Syllabus"),
                 "items": [
                     {
-                        "short_description": _("Create Your First Section and Subsection"),
-                        "long_description": _("Use your course outline to build your first Section and Subsection."),
+                        "short_description": _("Include Syllabus in Course Handouts"),
```

Comment:
if it's either a handout or on a separate page, maybe:
Include a detailed course syllabus